api/src/main/resources/db/changelog/database/tclass.xml

```diff
@@ -4,15 +4,15 @@
         xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.7
          http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.7.xsd">
 
-  <changeSet id="2" author="damian">
```

Comment:
tu tez zrobiłeś rename temu se to nie merguje. (merguje sie tylko zmian nie widac). Jak to zmerguje do main to t1.xml zmieni mi nazwe na tclass. Obczaj sobie jak działa git (to jedna z postawowych umiejetnoci które trzeba miec poza znajomoscią języka kodzenia)